app/src/components/MetricView.tsx

```diff
@@ -9,10 +9,10 @@ export interface MetricViewProps {
 
 export const MetricView = (props: MetricViewProps): React.ReactElement => {
   return (
-    <Box variant='metricCard' height='5rem' width='7rem'>
+    <Box variant='metricCard' height='7rem' width='10rem'>
```

Comment:
are you 100% sure all changes in this file are needed. you need to be fully sure because it will affect the CollectionPage